sys/include/net/gnrc/rpl/structs.h

```diff
@@ -238,7 +238,24 @@ struct gnrc_rpl_parent {
 typedef struct {
     uint16_t ocp;   /**< objective code point */
     uint16_t (*calc_rank)(gnrc_rpl_parent_t *parent, uint16_t base_rank); /**< calculate the rank */
-    gnrc_rpl_parent_t *(*which_parent)(gnrc_rpl_parent_t *, gnrc_rpl_parent_t *); /**< compare for parents */
+    gnrc_rpl_parent_t *(*which_parent)(gnrc_rpl_parent_t *, gnrc_rpl_parent_t *); /**< retrieve the better parent */
+
+    /**
+     * @brief   Compare two @ref gnrc_rpl_parent_t.
+     *
+     * This function is used to determine the parent list order. The parents
+     * are ordered from the preferred parent to the least preferred parent.
+     * This function should compare two parents based on the criteria of the
```

Comment:
the last sentence is rather vague (_should_?), be explicit: `Compares two parents based on the rank calculated by the objective function`. Btw. this sentence should be the first, followed by the other explaining the (possible) usage.